test/e2e/page-objects/cya/checkYourAppeal.js

```diff
@@ -0,0 +1,112 @@
+const DateUtils = require('utils/DateUtils');
+const haveAMRNContent = require('steps/compliance/have-a-mrn/content.en.json');
+const appointeeContent = require('steps/identity/appointee/content.en.json');
+const datesCantAttendContent = require('steps/hearing/dates-cant-attend/content.en');
+const oneMonthAgo = DateUtils.oneMonthAgo();
+const selectors = require('steps/check-your-appeal/selectors');
+const moment = require('moment');
+
+function enterDetailsFromStartToNINO() {
+
+    const I = this;
+
+    I.enterBenefitTypeAndContinue('PIP');
+    I.enterPostcodeAndContinue('WV11 2HE');
+    I.continueFromIndependance();
+    I.selectHaveYouGotAMRNAndContinue(haveAMRNContent.fields.haveAMRN.yes);
+    I.enterDWPIssuingOfficeAndContinue('1');
+    I.enterAnMRNDateAndContinue(oneMonthAgo);
+    I.selectAreYouAnAppointeeAndContinue(appointeeContent.fields.isAppointee.no);
+    I.enterAppellantNameAndContinue('Mr','Harry','Potter');
+    I.enterAppellantDOBAndContinue('25','01','1980');
+    I.enterAppellantNINOAndContinue('NX877564C');
+
+}
+
+function enterDetailsFromNoRepresentativeToSendingEvidence() {
+
+    const I = this;
+
+    I.selectDoYouHaveARepresentativeAndContinue('No');
+    I.enterReasonForAppealingAndContinue('A reason...');
+    I.enterAnythingElseAndContinue('Anything else...');
+    I.readSendingEvidenceAndContinue();
+
+}
+
+function enterDetailsFromNoRepresentativeToEnd() {
+
+    const I = this;
+
+    I.selectDoYouHaveARepresentativeAndContinue('No');
+    I.enterReasonForAppealingAndContinue('A reason...');
+    I.enterAnythingElseAndContinue('Anything else...');
+    I.readSendingEvidenceAndContinue();
```

Comment:
These four lines should be replaced with `I. enterDetailsFromNoRepresentativeToSendingEvidence `